Components migration docs (#716)
cloudposse/docs: docs/learn/maintenance/tutorials/how-to-update-components-yaml-to-new-organization.mdx

| vpc-peering                                     | github.com/cloudposse/terraform-aws-components.git//modules/vpc-peering?ref=\{\{.Version\}\}                                    | github.com/cloudposse-terraform-components/aws-vpc-peering//src?ref=\{\{.Version\}\}                                    |
| waf                                             | github.com/cloudposse/terraform-aws-components.git//modules/waf?ref=\{\{.Version\}\}                                            | github.com/cloudposse-terraform-components/aws-waf//src?ref=\{\{.Version\}\}                                            |
| zscaler                                         | github.com/cloudposse/terraform-aws-components.git//modules/zscaler?ref=\{\{.Version\}\}                                        | github.com/cloudposse-terraform-components/aws-zscaler//src?ref=\{\{.Version\}\}                                        |

> ### Special migration procedure
> 
> The following components have been splited into multiple repos by subcomponents.
> Please perform special migration procedures:

Reviewer: ```suggestion
### Special Migration Procedures

The following components have been split into multiple repos by subcomponents.

Here's what you need to know to update your configurations.

```

> #### spacelift

Reviewer: ```suggestion
#### Spacelift Components
```

> 1. Copy the `component.yaml` file from `spacelift` component to `spacelift/admin-stack`, `spacelift/spaces` and `spacelift/worker-pool` subcomponents dirs.
> 2. For each subcomponent in `component.yaml` update `source.version` to `1.532.0`
> 3. For each subcomponent in `component.yaml` update `source.uri` - replace `github.com/cloudposse/terraform-aws-components.git//modules/spacelift?ref=\{\{.Version\}\}` with

Reviewer: ```suggestion
1. Copy the `component.yaml` file from the `spacelift` component to the `spacelift/admin-stack`, `spacelift/spaces` and `spacelift/worker-pool` subcomponents directories.
2. For each subcomponent in the `component.yaml` update the `source.version` to `1.532.0`
3. For each subcomponent in the `component.yaml` update the `source.uri` to replace `github.com/cloudposse/terraform-aws-components.git//modules/spacelift?ref=\{\{.Version\}\}` with
```

| Component               | New URI                                                                                              |
|-------------------------|------------------------------------------------------------------------------------------------------|
| `spacelift/admin-stack` | `github.com/cloudposse-terraform-components/aws-spacelift-admin-stack//src?ref=\{\{.Version\}\}`     |
| `spacelift/spaces`      | `github.com/cloudposse-terraform-components/aws-spacelift-spaces//src?ref=\{\{.Version\}\}`          |
| `spacelift/worker-pool` | `github.com/cloudposse-terraform-components/aws-spacelift-worker-pool-asg//src?ref=\{\{.Version\}\}` |

> 4. Remove `component.yaml` file from `spacelift` component

Reviewer: ```suggestion
4. Remove the `component.yaml` file from the `spacelift` component
```

> #### tgw

Reviewer: ```suggestion
#### Transit Gateway Components (TGW)
```

| waf                                             | github.com/cloudposse/terraform-aws-components.git//modules/waf?ref=\{\{.Version\}\}                                            | github.com/cloudposse-terraform-components/aws-waf//src?ref=\{\{.Version\}\}                                            |
| zscaler                                         | github.com/cloudposse/terraform-aws-components.git//modules/zscaler?ref=\{\{.Version\}\}                                        | github.com/cloudposse-terraform-components/aws-zscaler//src?ref=\{\{.Version\}\}                                        |

> ### Special Migration procedure

Reviewer 2: ```suggestion
### Special migration procedure
```
